data/conf/logging/deb_logrotate.conf

```diff
@@ -6,6 +6,7 @@
         compress
         delaycompress
         sharedscripts
```

Comment:
hey Bas,

It appears that adm is not created on all debian versions. Can you add just the root user?

Thanks,
DanB

Reply:
No because the defaults for logrotate are root/root, and rsyslog uses root/adm by default.

Which Debian version are you encountering issues?

The oldest I have available is lenny, and its rsyslog also uses root/adm.

Reply:
An alternative may be to change the rsyslog configuration to use root/root for the cgrates log files.